.travis.yml

```diff
@@ -1,8 +1,18 @@
 script:
-  - docker build --no-cache --target test --tag todo-app:tests .
-  - docker run todo-app:tests tests/unit
-  - docker run todo-app:tests tests/integration
-  - docker run -e SECRET_KEY -e API_KEY -e API_TOKEN -e TODO_LIST_ID -e DOING_LIST_ID -e DONE_LIST_ID -e TRELLO_BOARD_ID todo-app:tests tests/e2e
+- echo $PASSWORD | docker login --username $USERNAME --password-stdin
+- docker build --no-cache --target test --tag todo-app:tests .
+- docker run todo-app:tests tests/unit
+- docker run todo-app:tests tests/integration
+- docker run -e SECRET_KEY -e API_KEY -e API_TOKEN -e TODO_LIST_ID -e DOING_LIST_ID -e DONE_LIST_ID -e TRELLO_BOARD_ID todo-app:tests tests/e2e
```

Comment:
Your e2e tests shouldn't need any of those list/board ids because it should be using a temporary board that it created itself.